tap_zendesk/discover.py

```diff
@@ -58,10 +58,18 @@ def discover_streams(client, config):
         streams.append({'stream': stream.name, 'tap_stream_id': stream.name, 'schema': schema, 'metadata': stream.load_metadata()})
```

Comment:
@prijendev If the stream would not have 'read' access, would we enter the JSON for that stream in the catalog?

Reply:
Yes, we enter the JSON for that stream in catalog